components/operator/Earthfile

```diff
@@ -89,6 +88,7 @@ deploy:
     END
     FROM --pass-args core+vcluster-deployer-image
     COPY --pass-args (+helm-update/helm) helm
+    RUN rm -rf helm/templates/gen/admissionregistration.k8s.io_v1_mutatingwebhookconfiguration_formance-system-mutating-webhook-configuration.yaml
```

Comment:
It's very precise, why deleting this file in particular ?

Reply:
Yes, the webhooks configuration of the operator doesn't works actually with our dev environment, so remove this file only for our deployment do the tricks (temporarily).
It was accidentally included in the previous PR in the generated chart on disk. I just moved the line where it does not impact production.